server.go

```diff
@@ -607,6 +637,23 @@ func (s *Server) removeConn(c io.Closer) {
 	}
 }
 
+func (s *Server) getCodec(contentSubtype string) (Codec, error) {
+	// TODO: do we really want to do string operations each time?
+	// This may be an extra allocation
+	// See comment in CustomCodecs
+	contentSubtype = strings.TrimSpace(strings.ToLower(contentSubtype))
+	if contentSubtype == "" {
```

Comment:
I was keeping these separate for now - you want some concept of a default codec anyways as this is what is currently implemented, and you might as well not do a map lookup.